tests/runtime/filter_lua.c

```diff
@@ -39,6 +39,13 @@ void set_output(char *val)
     pthread_mutex_unlock(&result_mutex);
 }
 
+void clear_output()
+{
+    pthread_mutex_lock(&result_mutex);
+    output = NULL;
+    pthread_mutex_unlock(&result_mutex);
+}
+
```

Comment:
This function is to remove following valgrind error.
It reports the pointer is already freed and it means the test uses previous output.

```
==19000== Invalid read of size 1
==19000==    at 0x4853A6E: strstr (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==19000==    by 0x148028: flb_test_type_int_key (filter_lua.c:223)
==19000==    by 0x145DDD: test_do_run_ (acutest.h:1007)
==19000==    by 0x146208: test_run_ (acutest.h:1178)
==19000==    by 0x14745B: main (acutest.h:1700)
==19000==  Address 0x54f9eb0 is 0 bytes inside a block of size 58 free'd
==19000==    at 0x484B27F: free (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==19000==    by 0x14AA46: flb_free (flb_mem.h:127)
==19000==    by 0x14F6D7: flb_lib_free (flb_lib.c:591)
==19000==    by 0x1489A0: flb_test_append_tag (filter_lua.c:388)
==19000==    by 0x145DDD: test_do_run_ (acutest.h:1007)
==19000==    by 0x146208: test_run_ (acutest.h:1178)
==19000==    by 0x14745B: main (acutest.h:1700)
==19000==  Block was alloc'd at
==19000==    at 0x4848899: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
==19000==    by 0x59F565: flb_malloc (flb_mem.h:80)
==19000==    by 0x5A38DD: out_lib_flush (out_lib.c:177)
==19000==    by 0x16DF42: output_pre_cb_flush (flb_output.h:528)
==19000==    by 0xAE19CA: co_init (amd64.c:117)
==19000== 
```